### PR TITLE
Fontbase 2.8.2 updated cask

### PR DESCRIPTION
**Pull request by Jarli01**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [ ] Named the cask according to the [token reference].
- [ ] `brew cask install {{cask_file}}` worked successfully.
- [ ] `brew cask uninstall {{cask_file}}` worked successfully.
- [ ] Checked there are no [open pull requests] for the same cask.
- [ ] Checked the cask was not [already refused].
- [ ] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/fontbase.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 cask 'fontbase' do
-  version '2.7.0'
-  sha256 '9dfed7a951855cf5cfedf7c42e1def948f6178a87cddf94e29d7b2764c724cb6'
+  version '2.8.2'
+  sha256 'e120cff005b12b8452e928bfd16cba7e535a7a2603d231ca30063952dfe9930c'
 
   url "https://releases.fontba.se/mac/FontBase-#{version}.dmg"
-  appcast 'https://fontba.se/_next/static/HBrfnSObRmIH0CqMQchqk/pages/downloads/thankyou.js'
+  appcast 'https://fontba.se/downloads/thankyou'
```
Comment (vitorgalvao) [inline Casks/fontbase.rb]: This page doesn’t show the version anywhere, thus it’s not suitable as an appcast. We need something like https://fontba.se/updates (but up to date, as that one only shows up to `2.8.0`).

Comment (Jarli01) [reply Casks/fontbase.rb]: Yeah I'm seeing that. . . so we can only update to 2.8.0 for the moment?

Comment (vitorgalvao) [reply Casks/fontbase.rb]: No, not at all. It just means we need a better appcast. I’ve also just noticed https://fontba.se/updates lists 2.8.0 as a June 14 2019 release, three days from now.

Comment (Jarli01) [reply Casks/fontbase.rb]: Eh. . . well ever download on their site, Windows, Mac and Linux all list as 2.8.2 lol. . .

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**